docs/_layouts/default.html

```diff
@@ -4,6 +4,13 @@
         {% include head.html %}
     </head>
     <body{% if page.slug %} class="{{ page.slug }}{% if page.slug != 'index-page' and page.slug !='how-it-works' and page.slug != 'guides' %} sub-page{% endif %}" {% endif %}>
+        {% include banner.html
+            text="✨&nbsp;&nbsp;Terragrunt v0.52.0 added support OpenTofu. To learn more, see the official"
+            link_url="https://github.com/gruntwork-io/terragrunt/releases/tag/v0.52.0"
+            link_text="release notes."
+            target="_blank"
+            has_button=false
+        %}
```

Comment:
I don't feel that strongly about this change, so I submit if for your consideration. It just makes the notice a little more concise. I was torn on using "Learn more" vs. "View release notes" for the link, but I think the clarity of the latter is nice.
```suggestion
        {% include banner.html
            text="✨&nbsp;&nbsp;Terragrunt v0.52.0 now supports OpenTofu."
            link_url="https://github.com/gruntwork-io/terragrunt/releases/tag/v0.52.0"
            link_text="View release notes"
            target="_blank"
            has_button=false
        %}
```

Reply:
I think this is better too - I was looking at it and thinking that it reads a little funky, but couldn't come up with better wording. Let's o with yours!